ui/__tests__/components/policy/req.test.js

```diff
@@ -23,13 +25,20 @@ describe('<Req />', () => {
     expect(hlText).toMatch(/123.45/);
   });
 
+  it('includes topiclinks with data when highlighted', () => {
+    req.topics = [{ id: 1, name: 'link 1' }, { id: 2, name: 'link 2' }];
+    const normalText = mount(
+      <Req highlighted href="" onClick={jest.fn()} req={req} />,
+    ).find('.topics').text();
+
+    expect(normalText).toMatch(/link 1/);
+    expect(normalText).toMatch(/link 2/);
+  });
+
   it('includes an anchor with the correct data', () => {
     const onClick = jest.fn();
-    const req = {
-      req_id: '1.1',
-      req_text: 'Text goes here',
-      topics: [],
-    };
+    req.req_id = '1.1';
+    req.req_text = 'Text goes here';
```

Comment:
What do you think about extending the `req` rather than modifying it?
```js
const modifiedReq = { ...req, req_id: '1.1', req_text: 'Text goes here' };
```
If we modify it in one test, the modified version will be shared by other tests, which makes the order of test execution matter.

Alternatively, we could use a `beforeEach`:
```js
describe('<Req />', () => {
    let req;
    beforeEach(() => {
        req = {
            req_id: '123.45',
            req_text: 'Some text here',
            topics: [],
            policy: {},
        };
    });

    it( ...
});
```